client/extensions/woocommerce/lib/countries/CA.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export default () => {
+	return (
+		{
+			code: 'CA',
+			name: translate( 'Canada' ),
+			states: [
+				{
+					code: 'AB',
+					name: translate( 'Alberta' ),
+				},
+				{
+					code: 'BC',
+					name: translate( 'British Columbia' ),
+				},
+				{
+					code: 'MB',
+					name: translate( 'Manitoba' ),
+				},
+				{
+					code: 'NB',
+					name: translate( 'New Brunswick' ),
+				},
+				{
+					code: 'NL',
+					name: translate( 'Newfoundland and Labrador' ),
+				},
+				{
+					code: 'NT',
+					name: translate( 'Northwest Territories' ),
+				},
+				{
+					code: 'NS',
+					name: translate( 'Nova Scotia' ),
+				},
+				{
+					code: 'NU',
+					name: translate( 'Nunavut' ),
+				},
+				{
+					code: 'ON',
+					name: translate( 'Ontario' ),
+				},
+				{
+					code: 'PE',
+					name: translate( 'Prince Edward Island' ),
+				},
+				{
+					code: 'QC',
+					name: translate( 'Quebec' ),
+				},
+				{
+					code: 'SK',
+					name: translate( 'Saskatchewan' ),
+				},
+				{
+					code: 'YT',
+					name: translate( 'Yukon Territory' ),
+				},
+			],
+			statesLabel: translate( 'Province' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'State/Province' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).